src/ContainerHeader.jsx

```diff
@@ -26,17 +28,35 @@ class ContainerHeader extends React.Component {
         }
     }
 
+    handleOwnerChange (value) {
+        this.setState({ owner: value });
+        if (this.props.onOwnerChanged) {
+            this.props.onOwnerChanged(value);
+        }
+    }
+
     handleFilterTextChange() {
         this.setState({ filterText: this.refs.filterTextInput.value }, this.filterChanged);
     }
 
     render() {
         return (
             <>
+                <label className="heading-label" htmlFor="containers-containers-filter">{_("Containers")}</label>
                 <Select.Select id="containers-containers-filter" initial={this.state.filter} onChange={this.handleFilterChange}>
-                    <Select.SelectEntry data='all'>{_("Everything")}</Select.SelectEntry>
-                    <Select.SelectEntry data='running'>{_("Images and running containers")}</Select.SelectEntry>
+                    <Select.SelectEntry data='running'>{_("Only running")}</Select.SelectEntry>
+                    <Select.SelectEntry data='all'>{_("All")}</Select.SelectEntry>
```

Comment:
Unrelated, but good change.

Reply:
Yeah, I also gave this select label that was not there before to be consistent with the new select. I can split it into separate commit, but I feel like not really needed. But if you think that would be better, then I'll happily do that.

Reply:
No worries, this was just thinking aloud.